src/coreComponents/schema/docs/VTKMesh.rst

```diff
@@ -14,6 +14,7 @@ partitionRefinement integer                                1         Number of p
 regionAttribute     string                                 attribute Name of the VTK cell attribute to use as region marker                                                                                                                                                                                                                          
 scale               R1Tensor                               {1,1,1}   Scale the coordinates of the vertices by given scale factors (after translation)                                                                                                                                                                                                
 translate           R1Tensor                               {0,0,0}   Translate the coordinates of the vertices by a given vector (prior to scaling)                                                                                                                                                                                                  
+useGlobalIds        integer                                1         If set to 1, mesh loading will use the GlobalId attribute for vertices and meshes, if available. This can avod unnecessary calculations. GlobalId arrays must be flagged as GLOBAL_IDS in the input mesh.
```

Comment:
Same, with the `avod` and generated file.
Maybe you did not regenerate the file after fixing the typo in your `registerWrapper`?

Reply:
Yep, I forgot to commit the auto-generated file, I will correct this in the next commit